cli/objects/jira_base.py

```diff
@@ -308,3 +308,11 @@ def _get_security_level_id(
             f"Security level {security_level} not found in {project_key}, no security level will be applied.",
         )
         return None
+
+    def get_issue_by_id_or_key(self, issue_id_or_key: str) -> Issue:
```

Comment:
the word `issue` in `issue_id_or_key` is redundant here, the function called `get_issue_by_id_or_key`. no need to repeat